jill/mirror.py

```diff
@@ -126,7 +127,7 @@ def pull_releases(self, *,
                              upstream=upstream,
                              overwrite=overwrite)
 
-
+# Carsten: TODO: shoudn't work anymore. Probably needs an option to change the VERSIONS_URL?
```

Comment:
If you feel it too much work to do here, it's okay for me to completely remove the `mirror` functionality; people could just use the standard s3 utils to mirror the binaries.

This was written back to when I didn't know that it's possible to do `s3 sync` :(

Reply:
Dropped it.